### ISSUE TITLE
Gradient Clipping

### ISSUE THREAD

**Issue by kellywzhang** [kellywzhang/reading-comprehension#2]
Need to add gradient clipping to RNN to prevent exploding gradients. Tensorflow has packages. Chen et al. has details regarding what gradient clipping value they used for the Stanford Reader.

**Comment by kellywzhang** [comment]
Implemented in train_model.py
